src/features/actorInfo/ActorInfo.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { fetchActorById, currentActorReset } from './actions';
+import { Header } from '../../components/Header';
+import { Footer } from '../../components/Footer';
+
+import styles from './ActorInfo.module.scss';
+
+const actorDetails = ({ name, imgUrl, biography }) => (
+  <article className={styles.actor}>
+    <div className={styles.actorInfo}>
+      <div className={styles.photoContainer}>
+        <img
+          className={styles.photo}
+          src={imgUrl}
+          alt={name} />
+      </div>
+      <div className={styles.textContent}>
+        <p className={styles.text}>
+          <span className={styles.paragraph}>Name: </span>
+          {name}
+        </p>
+        <p className={styles.text}>
+          <span className={styles.paragraph}>Biography: </span>
+          {biography}
+        </p>
+      </div>
+    </div>
+  </article>
+);
+
+class ActorInfoRoot extends Component {
+  componentDidMount() {
+    const { fetchActorById } = this.props;
+    const { id } = this.props.match.params;
+
+    fetchActorById(id);
+  }
+
+  componentWillUnmount() {
+    const { currentActorReset } = this.props;
+
+    currentActorReset();
+  }
+
+  render() {
+    const { currentActor } = this.props;
+
+    if (!currentActor) {
+      return <h1>Loading...</h1>
+    }
+
+    return (
```

Comment:
You can use Loading directly in main render:
![image](https://user-images.githubusercontent.com/3533737/76350981-58976000-6315-11ea-9341-0d546b83cde3.png)